Sources/Controllers/DashboardOnMap/MapSettings/OAMapSettingsMainScreen.mm

```diff
@@ -121,321 +361,26 @@ - (NSString *)getMapLangValueStr
         case 6: // LocalizedOrTransliterated
             languageValue = [NSString stringWithFormat:@"%@ (%@)", prefLang, [OALocalizedString(@"sett_lang_trans") lowercaseStringWithLocale:[NSLocale currentLocale]]];
             break;
-            
+
         default:
             break;
     }
-    
+
     return languageValue;
 }
 
 - (NSArray *) getAllCategories
 {
-    NSMutableArray *res = [NSMutableArray array];
+    NSMutableArray *res = [NSMutableArray new];
```

Comment:
Both variants of initialization are acceptable. No need to change one for the other. (I personally prefer using array/dictionary initialization though it's just a matter of personal preference)